medical_prescription_state_verify/models/medical_prescription_order.py

```diff
@@ -18,29 +18,16 @@ class MedicalPrescriptionOrder(models.Model):
             that has been defined in `_ALLOWED_CHANGE_STATES`)
         _ALLOWED_CHANGE_STATES: `list` of keys defining status types in which
             an RX can be moved to after being verified
+            4 - Verified
+            5 - Cancelled
+            6 - Exception
     """
 
-    _inherit = 'medical.prescription.order'
+    _inherit = ['medical.prescription.order', 'base.kanban.abstract']
```

Comment:
Why inheriting kanban? This should be handled with a dependency to the prescription stage module